boards/arm/nrf52810_pca10040/Kconfig

```diff
@@ -11,4 +11,14 @@ config BOARD_ENABLE_DCDC
         select SOC_DCDC_NRF52X
         default y
 
+if BT
+
+# BT_CTLR depends on BT. When BT is enabled we should default to also
+# enabling the controller.
+config BT_CTLR
+	default y
```

Comment:
Could also do `default y if BT` by the way, which saves some lines. No super strong opinions though.

Reply:
agree, no need of if BT, use default y if BT.

Reply:
Done